modules/libcom/test/epicsStringTest.c

```diff
@@ -195,6 +195,32 @@ void testStrTok(void)
     testTok(NULL, " \t", "bbb ", "bbb", "");
 }
 
+static
+void testEpicsStrPrintEscaped(void)
+{   
+    const char *filename = "testEpicsStrPrintEscaped";
+    // Create a file then re-open it read only
+    FILE *readOnly = fopen(filename, "a");
+    fclose(readOnly);
+    readOnly = fopen(filename, "r");
+
+    testDiag("testEpicsStrPrintEscaped()");
+
+    // Passing cases
+    testOk1(epicsStrPrintEscaped(stdout, "1234", 4) == 4);
+    testOk1(epicsStrPrintEscaped(stdout, "\a\b\f\n\r\t\v\\\'\"", 10) == 20);
```

Comment:
I didn't notice that CI had not run.  There is an issue in the test output with the special characters confusing things.

```
...
ok 439 - inp: "(nil)" delim: "  " tok: "bbb"=="bbb" save: ""==""
# testEpicsStrPrintEscaped()
1234ok 440 - epicsStrPrintEscaped(stdout, "1234", 4) == 4
\a\b\f\n\r\t\v\\\'\"ok 441 - epicsStrPrintEscaped(stdout, "\a\b\f\n\r\t\v\\\'\"", 10) == 20
ok 442 - epicsStrPrintEscaped(readOnly, "1234", 4) == -1
...
```

results in

```
  Test Summary Report
  -------------------
  epicsStringTest.tap           (Wstat: 0 Tests: 444 Failed: 0)
    Parse errors: Tests out of sequence.  Found (442) but expected (440)
                  Tests out of sequence.  Found (443) but expected (441)
                  Tests out of sequence.  Found (444) but expected (442)
                  Tests out of sequence.  Found (445) but expected (443)
                  Tests out of sequence.  Found (446) but expected (444)
  Displayed the first 5 of 6 TAP syntax errors.
  Re-run prove with the -p option to see them all.
  Files=49, Tests=4376,  0 wallclock secs ( 0.17 usr  0.05 sys +  0.05 cusr  0.01 csys =  0.28 CPU)
  Result: FAIL
````